### PR TITLE
External

### PR DESCRIPTION
**Pull request by kreier**
Create a different website about this project.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,617 @@
...
+
+The wind generator arrived just one day later! Looked at location on top of the roof, 6th floor in Nha Be. Empty room for equipment is there, rain proved, and space for the 5 wired from solar and wind to the control unit. Maybe next week start first test setup?
+
+![Wind generator](pic/2020-05-08_wind.jpg)
+
+Data: __JLS-500__ with 24V and 500 W
+
+> 2020/05/14
+
+The TTGO can be programmed with MicroPython and [devbis](https://github.com/devbis) created both a [slow python driver](https://github.com/devbis/st7789py_mpy/) for the display ST7789 as well as a [fast C variant](https://github.com/devbis/st7789_mpy). Further description of this module [here](https://sites.google.com/site/jmaathuis/arduino/lilygo-ttgo-t-display-esp32). And the tft driver from Loboris is working as well, details in [this instuctable from February 2020](https://www.instructables.com/id/TTGO-color-Display-With-Micropython-TTGO-T-display/). Original data at [LilyGo github](https://github.com/Xinyuan-LilyGO/TTGO-T-Display).
```
Comment (kreier) [inline docs/index.md:355]: This actually writess about the TTGO T-Display. The company has other boards to offer!
TTGO_ESP32

File: docs/_config.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+theme: jekyll-theme-cayman
+google_analytics: [UA-126581035-2]
```
Comment (kreier) [inline docs/_config.yml:2]: That's my personal code.

**Review by aisvn-data — APPROVED** [review]
Looks legit

**Review by kreier — COMMENTED** [review]
Please extend the pictures and recent data.

File: docs/index.md (hunk 1 above)
Comment (aisvn-data) [reply docs/index.md:355]: Working on it. First need the folder for all the pictures.

File: docs/_config.yml (hunk 2 above)
Comment (aisvn-data) [reply docs/_config.yml:2]: Please double check if it counts the views. And I guess Github pages is not yet activated.

**Comment by aisvn-data** [comment]
Leave the branch open for further contribution.

**Comment by kreier** [comment]
Should be fine now, other parts will be updated later!

**Pull request merged**